lte/gateway/c/oai/lib/mobility_client/MobilityClientAPI.cpp

```diff
@@ -43,7 +43,7 @@ using magma::lte::MobilityServiceClient;
 extern task_zmq_ctx_t spgw_app_task_zmq_ctx;
 
 static itti_sgi_create_end_point_response_t handle_allocate_ipv4_address_status(
-    const grpc::Status& status, struct in_addr inaddr, const char* imsi,
+    const grpc::Status& status, int vlan, struct in_addr inaddr, const char* imsi,
     const char* apn, const char* pdn_type,
     itti_sgi_create_end_point_response_t sgi_create_endpoint_resp);
```

Comment:
Can you change the signature of this file so that `sgi_create_endpoint_resp` is passed by value and return is either int or void?

Reply:
I will send separate patch for this.